rockspec/lua-resty-hmac-ffi-1.0-0.rockspec

```diff
@@ -0,0 +1,24 @@
+package = 'lua-resty-hmac-ffi'
+version = '1.0-0'
+
+source = {
+  url = 'git://github.com/jamesmarlowe/lua-resty-hmac',
+  tag = 'v1.0'
+}
+
+description = {
+  summary = 'Lua driver for making and receiving hmac signed requests',
+  homepage = 'https://github.com/jamesmarlowe/lua-resty-hmac',
```

Comment:
I think it should be `https://github.com/jkeys089/lua-resty-hmac`, right?

Reply:
sorry for this, fixed right now